client/my-sites/scan/main.tsx

```diff
@@ -185,7 +185,7 @@ class ScanPage extends Component< Props > {
 				) }
 				<p>
 					{ translate(
-						'We will send you an email once the scan completes, in the meantime feel ' +
+						'We will send you an email if security threats are found, in the meantime feel ' +
```

Comment:
There are 2 comma splices here. Should be:

```
We will send you an email if security threats are found. In the meantime feel free to continue to use your site as normal; you can check back on progress at any time.
```

Reply:
Agreed with what @singerb said above. We could also use the same structure as the first lines on this PR and go for something like this:

> We will send you an email if Jetpack Scan finds any security threats. In the meantime feel free to continue to use your site as normal, you can check back on scanning progress at any time.

Reply:
Thanks, fixed